Packs/AnomaliSecurityAnalyticsAlerts/Integrations/AnomaliSecurityAnalyticsAlerts/AnomaliSecurityAnalyticsAlerts.py

```diff
@@ -256,32 +260,100 @@ def command_update_alert(client: Client, args: dict) -> CommandResults:
     Returns:
         CommandResults.
     """
-    status = str(args.get('status'))
-    comment = str(args.get('comment'))
-    uuid_val = str(args.get('uuid'))
-    if uuid_val == 'None':
+    uuid_val = str(args.get("uuid"))
+    if not uuid_val or uuid_val.lower() == "none":
         raise DemistoException("Please provide 'uuid' parameter.")
-    if status == 'None' and comment == 'None':
-        raise DemistoException("Please provide either 'status' or 'comment' parameter.")
-    columns = {}
-    if status != 'None':
-        columns['status'] = status
-    if comment != 'None':
-        columns['comment'] = comment
-    data = {
-        "table_name": "alert",
-        "columns": columns,
-        "primary_key_columns": ["uuid_"],
-        "primary_key_values": [[uuid_val]]
-    }
+
+    supported_fields = ["status", "comment", "assignee", "owner", "severity"]
+    columns = {field: args[field] for field in supported_fields if args.get(field) and str(args.get(field)).lower() != "none"}
+
+    if not columns:
+        raise DemistoException(f"No valid fields provided to update. Supported fields are: {', '.join(supported_fields)}")
+
+    data = {"table_name": "alert", "columns": columns, "primary_key_columns": ["uuid_"], "primary_key_values": [[uuid_val]]}
+
     response = client.update_alert(data)
+
+    readable = tableToMarkdown(name="Alert Updated Successfully", t=columns, removeNull=True)
+
     return CommandResults(
-        outputs_prefix='AnomaliSecurityAnalytics.UpdateAlert',
-        readable_output=tableToMarkdown(name="Update Alert", t=response, removeNull=True),
-        raw_response=response
+        outputs_prefix="AnomaliSecurityAnalytics.UpdateAlert",
+        outputs_key_field="uuid",
+        outputs={"uuid": uuid_val, "updated_fields": columns},
```

Comment:
in the YML the context path for this command are 
`AnomaliSecurityAnalytics.UpdateAlert.message`, are `message` still output in the corrent implementation?
in addition, please update the context output with the added keys